_data/simple-icons.json

```diff
@@ -3725,6 +3725,11 @@
             "hex": "0018A8",
             "source": "https://www.db.com"
         },
+        {
+            "title": "Deutsche Post",
+            "hex": "000000",
```

Comment:
```suggestion
            "hex": "FFCC00",
```

Reply:
I did some research and finally found the brand guidelines that I missed on the weekend (will add the link to the JSON as well): https://www.dpdhl-brands.com/de/dp/logo

According to the guidelines, the color of the posthorn icon may not be changed from black.

I saw your comment in the other PR that the brand color is not the same as the icon color. However many uses of Simple Icons blindly color the SVG with the brand color. Which is correct for most icons, but breaks the guidelines here.

How can we solve this dilemma?

Reply:
@lukasbestle, we put the onus on our users to ensure they read and adhere to all branding & trademark guidelines we provide. As you can see from the [colour section](https://www.dpdhl-brands.com/de/dp/farben) of their brand guidelines, the primary _brand_ colour (which is what we aim to include in our data) here is `#ffcc00`.